torch/nn/functional.py

```diff
@@ -3672,35 +3672,18 @@ def upsample(input, size=None, scale_factor=None, mode="nearest", align_corners=
 
 upsample.__doc__ = upsample.__doc__.format(**reproducibility_notes)
 
+_InterpolateSizeType = Union[None, int, Tuple[int], Tuple[int, int], Tuple[int, int, int]]
```

Comment:
Is `Union[None, ...]` exactly the same as `Optional[...]` ?

Reply:
If the ellipsis stands for one item, yes. We still need `Union` here. I can replace with more explicit `Optional[Union[int, Tuple[int], Tuple[int, int], Tuple[int, int, int]]]`, is it better?